src/PowerBIEmbedded/PowerBI/ChangeLog.md

```diff
@@ -18,6 +18,7 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Removed workspace collection cmdlets
```

Comment:
Hey @ayeshurun is there an announcement can be link here?

Reply:
Also, when exactly will the function be deprecated? By "deprecated" do you mean they will stop working or not?

We still see quite some usage of the cmdlets, so we can't just remove the cmdlets while they still work.

Reply:
@isra-fel The intention is to remove them, as this feature is not available anymore. We shouldn't see any usage.

Reply:
> @isra-fel The intention is to remove them, as this feature is not available anymore. We shouldn't see any usage.

If the customer call the cmdlets that are removed in your PR, what will happen for now?